.gitignore

```diff
@@ -13,3 +13,4 @@ coverage
 build
 artifacts
 .idea
+.vscode
```

Comment:
VS Code creates this folder.  I added it here so VS Code users don't accidentally commit this folder to the repo.